### PR TITLE
Esperanto translation

### PR DESCRIPTION
**Pull request by KuboF**
## Description
Esperanto translation of Kimai. Translated by Sofia Zaretskaia, reviewed by Michal Matúšov (@KuboF) for E@I.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist
- [x] I verified that my code applies to the guidelines (`composer kimai:code-check`)
- [x] I updated the documentation (see [here](https://github.com/kimai/www.kimai.org/tree/master/_documentation))
- [x] I agree that this code is used in Kimai and will be published under the [MIT license](https://github.com/kevinpapst/kimai2/blob/master/LICENSE)

### CODE REVIEW

**Comment by kevinpapst** [comment]
@KuboF I have pushed some changes, to support Esperanto (where possible) in the frontend tools. The `fullcalendar` doesn't know the `eo` locale, so it falls back to `en`. Same goes for the selectpicker `select2`.

Select2 is very simple to translate, just a few phrases (as example the [english file](https://github.com/select2/select2/blob/develop/src/js/select2/i18n/en.js)).
Just if you feel inspired to translate it as well...

Anyway: Can you please do a test-run locally with the changes?

**Comment by KuboF** [comment]
Thanks for rebuild, @kevinpapst!

My local test works well!

I have found some translation problems - now fixed with the 2 new commits.

Esperanto translations / locales for dependencies:

- 'select2': https://github.com/select2/select2/pull/5823 (open)
- 'fullcalendar': translation prepared, I will PR after merge of my previous PR (I missed my branches a bit)

**Comment by kevinpapst** [comment]
Kimai is still using Fullcalendar v3 and there seems to be a version v5 coming soon. 
Not sure which version you translated. But its great to see that you are actively working on raising awareness for Esperanto 👍 

Anyways - let me know when you are done with the PR, then I will merge it. 
Changes can be done any time later, I want you to take the time needed for final checks.

File: translations/export.eo.xlf [hunk 1]
```diff
@@ -0,0 +1,16 @@
+
```
Comment (kevinpapst) [inline translations/export.eo.xlf:1]: please remove the new line, it is invalid XML otherwise and the build fails

Comment (KuboF) [reply translations/export.eo.xlf]: Fixed. Thanks for info!

**Pull request merged**